CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to CRA Template Truemark
+
+Thank you for your interest in contributing to our project! We appreciate your interest in contributing.
+
+## How to Contribute
+
+- Fork the Template: On GitHub, click "Fork" to create your own copy.
+- Clone the forked repository to your local machine.
+
+    `git clone https://github.com/TruemarkDev/cra-template-truemark.git` for https.
+
+    `git clone git@github.com:TruemarkDev/cra-template-truemark.git` for ssh.
+
+- Install dependencies using `npm install`.
+- Start the project using `npm start`.
+- Check out to a new branch.
+- Make your changes and commit them.
+- Push your changes to repository.
+- Create a pull request with a clear title and description.
+
+## Code Guidelines
+
+- Follow consistent coding style.
+- Write clear and meaningful commit messages.
+
```

Comment:
Let's add another point, so that when user change anything related to library then s/he will update documentation as well.
- Update documentation for the changes made if needed.